test/router.test.ts

```diff
@@ -0,0 +1,12 @@
+import indefinite from 'indefinite';
+import createRouter, { Routes } from '../src/router';
+
+describe('router', (): void => {
+  const router = createRouter();
+
+  Object.entries(Routes).forEach(([key, value]) => {
+    it(`has ${indefinite(key)} route`, (): void => {
+      expect(router.route(value)).toBeTruthy();
+    });
+  });
```

Comment:
Feels... bit dirty.

Reply:
Which bit? ... The loop around the `it` is pretty standard but yes looks a tad ugly compared to something like python. The check that is truthy is pretty vague - did you want to tighten it? for example check its a function?

Reply:
Rephrase to `has a route named|for ${key}` and avoid a 156KB library?
In general for a unit test it's unexpected to use test code that is vastly more complex than the unit under test, it may break more often that what it is testing, as encountered in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/40238